packages/react-native-worklets/src/shareables.ts

```diff
@@ -355,6 +372,26 @@ function clonePlainJSObject<T extends object>(
   return clone;
 }
 
+function cloneTurboModule<T extends object>(
+  value: T,
+  shouldPersistRemote: boolean,
+  depth: number
+): ShareableRef<T> {
+  const clonedProto = Object.getPrototypeOf(value);
```

Comment:
Please also add a comment why we **don't** call `makeSharableClone` on `proto`.
```suggestion
  const proto = Object.getPrototypeOf(value);
```